dynamic_programming/tribonacci.py

```diff
@@ -0,0 +1,25 @@
+# Tribonacci sequence using Dynamic Programming
+
+
+def tribonacci(num: int) -> list:
+    """
+    Given a number, return first n Tribonacci Numbers.
+    >>> tribonacci(5)
+    [0, 0, 1, 1, 2]
+    >>> tribonacci(8)
+    [0, 0, 1, 1, 2, 4, 7, 13]
+    """
+    dp = [0 for i in range(num)]
+    dp[0] = dp[1] = 0
```

Comment:
```suggestion
```
This isn't really necessary because the list `dp` is already instantiated with all 0's